cl/recap/management/commands/pacer_iquery_scraper.py

```diff
@@ -15,6 +15,18 @@
 from cl.search.models import Docket, Court
 
 
+def get_docket_ids_missing_info(num_to_get):
+    docket_ids = set()
+    docket_ids.update(
+        Docket.objects.filter(
+            date_filed__isnull=True, source__in=Docket.RECAP_SOURCES
+        )
+        .order_by("-view_count")[:num_to_get]
+        .values_list("pk", flat=True)
+    )
+    return docket_ids
```

Comment:
This might fail Black, but what about:

```suggestion
    return set(
        Docket.objects.filter(
            date_filed__isnull=True, source__in=Docket.RECAP_SOURCES
        )
        .order_by("-view_count")[:num_to_get]
        .values_list("pk", flat=True)
    )
```

Reply:
Passes black, seems fine